mobileapps/views.py

```diff
@@ -1129,3 +1129,37 @@ def delete(self, request, theme_id):
         return Response(status=status.HTTP_204_NO_CONTENT)
 
 
+class OrganizationThemeRemoveImageView(MobileRetrieveUpdateDestroyAPIView):
+    """
+    **Use Case**
+
+        Remove logo image or header background image from the theme.
+
+    **Example Requests**
+
+        DELETE /api/server/mobileapps//themes/{id}/remove/{attribute}
+
+        - attribute
+            - Logo Image: logo_image
+            - Header Background Image: header_bg_image
+    """
+
+    serializer_class = ThemeSerializer
+    lookup_url_kwarg = 'theme_id'
+
+    def __init__(self):
+        self.permission_classes += (IsStaffOrReadOnlyView,)
```

Comment:
Can we use a permission witch checks only if request user is staff?. I mean we don't have any readonly view here.